### ISSUE TITLE
msal-browser does not get cached login when the page refreshes

### ISSUE THREAD

**Issue by SihirliDev** [AzureAD/microsoft-authentication-library-for-js#2551]
>> Please follow the issue template below. Failure to do so will result in a delay in answering your question.

## Library
- [ ] `msal@1.x.x` or `@azure/msal@1.x.x`
- [x] `@azure/msal-browser@2.4.1` and above
- [ ] `@azure/msal-angular@0.x.x`
- [ ] `@azure/msal-angular@1.x.x`
- [ ] `@azure/msal-angularjs@1.x.x`

**Important**: Please fill in your exact version number above, e.g. `msal@1.1.3`.

## Framework
React

## Description
I'm using msal-browser in React. When using redirect flow, the user login is cached in the browser (localStorage or sessionStorage does not matter) however when the page is refreshed the msal-browser component does not read this cached credential and redirects user to the azure login page (and automatically redirected back).

This error is seen in versions 2.4.1 and above. No error in 2.4.0. Probably something changed in the BrowserStorage mechanism when upgrading from 2.4.0

## Security
- [ ] Is this issue security related?

## Regression
- [x] Did this behavior work before? 
Version: 2.4.0 and below

## MSAL Configuration

```js
// Provide configuration values here.
// For Azure B2C issues, please include your policies.
```

## Reproduction steps
```js
// Provide relevant code snippets here.
// For Azure B2C issues, please include your policies.
```

## Browsers/Environment
- [x] Chrome
- [ ] Firefox
- [ ] Edge
- [ ] Safari
- [ ] IE
- [ ] Other (Please add browser name here)

**Comment by jasonnutter** [comment]
@SihirliDev This appears similar to #2524, which is fixed by #2516.

**Comment by SihirliDev** [comment]
Thank you @jasonnutter I'll check it.

**Comment by SihirliDev** [comment]
This is fixed in v2.6.0. Thank you.
